src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -404,7 +406,7 @@ export default class OfficeAddresses extends Mixins(CommonMixin) {
    * Sets local address data and "inherit" flags from store.
    */
   private setLocalProperties (): void {
-    if (this.getOfficeAddresses.registeredOffice) {
+    if (this.getOfficeAddresses?.registeredOffice) {
```

Comment:
This should not be necessary, since `state.stateModel.officeAddresses` is initialized as `{}`.

But this raises the question, should `officeAddresses` be initialized to null or an object with empty fields instead? It depends how it's used. It may not matter here. FWIW, both are used in Create UI:

https://github.com/bcgov/business-create-ui/blob/7404fcbbccccced5736ad316294daa86b80153a9/src/store/state/state-model.ts#L33

https://github.com/bcgov/business-create-ui/blob/7404fcbbccccced5736ad316294daa86b80153a9/src/store/state/state-model.ts#L168